doc/source/whatsnew/v0.25.0.rst

```diff
@@ -123,13 +123,13 @@ If installed, we now require:
 +=================+=================+==========+
 | beautifulsoup4  | 4.4.1           |          |
 +-----------------+-----------------+----------+
-| openpyxl        | 2.2.6           |          |
+| openpyxl        | 2.4.0           |          |
 +-----------------+-----------------+----------+
-| pymysql         | 0.6.6           |          |
+| pymysql         | 0.7.9           |          |
```

Comment:
We don't actually require these versions for sqlalchemy / pymsql, they are just the recommended ones. So may want to re-word this (can do in a followup). We are don't actually reject an earlier version.